app/3.0/docs/tools/polymer-cli-commands.md

```diff
@@ -277,9 +233,11 @@ Sets npm mode. Dependencies are installed from npm, the component directory is s
 polymer serve --npm --module-resolution="node"
 ```
 
+Because the dependencies for Polymer 3.0 projects are defined in `package.json`, the `--npm` flag is usually required. You will probably find it easier to [add this option to your polymer.json configuration](polymer-json).
```

Comment:
I don't know how we want to phrase this, but it appears that this should not be required for most cases anymore.

(It's only required if they're using npm, and using old-style element project URLs that access peer dependencies using '../' instead of by name, like '@polymer/polymer/...')

Reply:
I am not clear on what this flag does or when to use it :/ here is a guess: 

`--npm` is required when: 

* you are using npm to install dependencies, and
* you are importing dependencies by path (e.g. `'../node_modules/@polymer/...'`) instead of by module specifiers

If you are using npm and importing by module specifiers ('@polymer/polymer/...'), you don't need this flag.